library/alloc/src/vec/is_zero.rs

```diff
@@ -49,6 +53,8 @@ unsafe impl<T> IsZero for *mut T {
     }
 }
 
+/************ Arrays ************/
```

Comment:
I'm not personally a fan of comments like this -- it's not clear that section headers are really all that helpful and they make it harder to move things around down the line.

Reply:
Removed those.